.circleci/config.yml

```diff
@@ -38,6 +40,14 @@ jobs:
           name: Copy_Artifacts
           command: |
             . .circleci/copy_artifacts.sh
+      - run:
+          name: Build_Library (with OpenSSL 1.1)
+          command: |
+            . .circleci/build_lib.sh Release $CORE_SSL_1_1
+      - run:
+          name: Copy_Artifacts (with OpenSSL 1.1)
+          command: |
+            . .circleci/copy_artifacts.sh Release $CORE_SSL_1_1
```

Comment:
Here you should give the `copy_artifacts.sh` the `CORE_SSL_1_1` as first argument so `copy_artifacts.sh` will set `TARGET` properly.

Reply:
Good catch.